src/rgw/setup/rgw.py

```diff
@@ -544,7 +544,16 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
             #    Log.info(f'User creation is successful on "{Rgw._machine_id}" node.')
             #    Rgw._set_consul_kv(rgw_consul_idx, const.CONSUL_LOCK_KEY, const.ADMIN_USER_CREATED)
             # else:
-            machine_ids = Rgw._get_cortx_conf(conf, const.MACHINE_IDS_KEY)
+            storage_set = Rgw._get_cortx_conf(conf,\
+                const.STORAGE_SET % Rgw._machine_id)
+            storage_set_count = Rgw._get_cortx_conf(conf,\
+                const.STORAGE_SET_COUNT)
+            machine_ids = []
+            for storage_set_index in range(0, storage_set_count):
+                if Rgw._get_cortx_conf(conf,\
+                    const.STORAGE_SET_NAME % storage_set_index) == storage_set:
+                    machine_ids = Rgw._get_cortx_conf(conf,\
```

Comment:
Should this get appended to 'machine_ids' list instead of reassignment?

Reply:
Here we can't use appended since this "Rgw._get_cortx_conf(conf,const.STORAGE_SET_NODE % storage_set_index)" itself returns a list, we can use extend instead but existing code server the same.